src/index.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+import React, {PropTypes} from 'react';
+import {Hermes} from './hermes';
+import {createStore, bindActionCreators} from '@streammedev/flux-store';
+import valueReducer, {changeValue} from './actions/value';
```

Comment:
this could be `import createReducer from "./reducers/index";`, with an index reducer in lieu of storing reducers in actions. More of a would like to see than a requirement, especially with all of the fancy es6 syntactic sugar.